plugins/firebase.js

```diff
@@ -59,6 +59,75 @@ const fireDb = {
     }
     return introTexts
   },
+  getEvents: async () => {
+    const websites = await fireDb.getWebsites()
+    const events = {}
+    for (const website of websites) {
+      const websiteData = await db
+        .collection(webCollection)
+        .doc(website)
+        .collection('Events')
+        .get()
+      events[website] = await websiteData.docs.map(doc => {
+        const data = doc.data()
+        return {
+          id: doc.id,
+          title: data.title,
```

Comment:
don't you need || '' here as well?

Reply:
The || "" is in case there aren't those values in the firebase. There are inconsistencies between event fields for LHD and NWPlus, so in case the field does not exist, then it will be set to an empty string instead of undefined

Reply:
OK sounds nice